settings.gradle

```diff
@@ -1,6 +1,7 @@
 rootProject.name = 'opentelemetry-java-contrib'
 
 include ':example'
+include ':jmx-metrics'
```

Comment:
I cloned this branch and opened it in IntelliJ, and things appear to be mostly fine, but `contrib/jmx-metrics` isn't recognized out of the box as a project module by IntelliJ even though its sibling directory, `contrib/example`, is. Not sure if this is a quirk in IntelliJ and/or if there's a Gradle (which I'm not an expert in) config that can be tweaked.

Reply:
I've found enabling auto reload can be helpful, but haven't had any issues personally (can run all expected tasks): https://www.jetbrains.com/help/idea/work-with-gradle-projects.html#auto_reload

Any errors or logs pop up?

Reply:
That fixed it, thanks!